File: core-java-practice/gcr-codebase/programming-elements/CalculateSimpleInterest.java
package Dayone;

import java.util.Scanner;

public class CalculateSimpleInterest {
  public static void main(String [] args){
    // write a prgm to calculate the simple interest 
        Scanner sc = new Scanner(System.in);
        double p = sc.nextDouble();
        double r = sc.nextDouble();        
        double t = sc.nextDouble();        
        // Calculate simple interest
        double simpleInterest = (p * r * t) / 100;
        // Print the simple interest
        System.out.println("Simple Interest is " + simpleInterest);
    }
}

File: core-java-practice/gcr-codebase/programming-elements/CalculateSimpleInterest.cs
using System;
namespace ProgrammingElements{
    class CalculateSimpleInterest{
        static void Main(string[] args){
            // write a program to calculate the simple interest
            double p = double.Parse(Console.ReadLine());
            double r = double.Parse(Console.ReadLine());
            double t = double.Parse(Console.ReadLine());
            // calculate simple interest
            double simpleInterest = (p * r * t) / 100;
            // print the simple interest
            Console.WriteLine("Simple Interest is " + simpleInterest);
        }
    }
}
